docs/api.rst

```diff
@@ -25,6 +25,5 @@
 API Docs
 ========
 
-es_jsonschema
+domapping
```

Comment:
We'll need to plug autodoc here, but that can be done later before making the release... This PR is only about renaming.